### ISSUE TITLE
[Github Issues Log] Uncaught Exception processing issue webhook

### ISSUE THREAD

**Issue by b2z** [joomla/jissues#897]
Github Issues Log is throwing Critical errors:

> Uncaught Exception processing issue webhook {"exception":"[object] (RuntimeException(code: 0): JTracker\Database\AbstractDatabaseTable::load can not bind. at

**Comment by zero-24** [comment]
Is this a cutted amessage as it sops with 'at'? ;)

**Comment by mbabker** [comment]
No, that's all the message was.  Until https://github.com/joomla/jissues/commit/05b98ac71c9196820f6ef7ab0afa221262080683

**Comment by mbabker** [comment]
Something's handling invalid data.  It's erroring out trying to bind a key value of `210083` (as an example) to the issues table.  None of the issues have six digit numbers, either as a local ID or GitHub.  Our activities table isn't up to that yet (`183503` is the last ID there).  It's not a GitHub activity ID; those are in the millions.

**Comment by mbabker** [comment]
I'm now thoroughly confused.  The webhook opening #903 failed with this same type of error.  See https://gist.github.com/mbabker/9466b20ef96362ffa364c302c11ec27c for the hook and response.  The ID it's groaning about in the error response isn't part of the webhook payload.

**Comment by mbabker** [comment]
Figured this one out.  When I opened #904 another message got added to the log about not being able to bind ID `212411`.  That's the ID of the open activity for that record.  So our code at https://github.com/joomla/jissues/blob/master/src/App/Tracker/Controller/Hooks/ReceiveIssuesHook.php#L147-L170 has issues when activity records are inserted after the issue record then we try to create an `IssuesTable` instance relying on the last insert ID from the database.
